app/components/table_cell_component.rb

```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
 
 class TableCellComponent < ViewComponent::Base
-  def initialize(container_link: nil)
+  def initialize(container_link: nil, classes: 'table-cell')
```

Comment:
nit thing, but can we call this `class_names`.

Reply:
super reasonable, updating now.